### PR TITLE
First ever contribution,window react twitter change

### PR DESCRIPTION
**Pull request by him157**
I just knew sometime in the future this will probably be put in,so I guess I came to contribute some small changes in the code,because why not?

### CODE REVIEW

**Comment by him157** [comment]
dammit,I knew something is not right,but where is it then... I really need to work harder before even think about contribute don't I.

**Comment by him157** [comment]
pffffffff oh come on,so it was just some little thing? I thought it was something big that I messed up,well the check passed at least

File: Monika After Story/game/zz_windowreacts.rpy [hunk 1]
```diff
@@ -394,3 +394,44 @@ label monika_monikamoddev:
...
+            eventlabel="monika_monikatwitter",
+            category=['twitter', 'lilmonix3'],
+            rules={"notif-group": "Window Reactions", "skip alert": None},
+            show_in_idle=True
+        ),
+        code="WRS"
+    )
+
+label monika_monikatwitter:
+    call display_notif(m_name, "so you are checking my twitter ,\nsee anything you want to talk about [player]?",'Window Reactions')
```
Comment (multimokia) [inline Monika After Story/game/zz_windowreacts.rpy]:
```suggestion
    call display_notif(m_name, "I see you're checking my twitter, [player].\nSee anything you want to talk about?",'Window Reactions')
```

File: Monika After Story/game/zz_windowreacts.rpy [hunk 2]
```diff
@@ -394,3 +394,44 @@ label monika_monikamoddev:
     if not _return:
         $ mas_unlockFailedWRS('monika_monikamoddev')
     return
+    
+init 5 python:
+    addEvent(
+        Event(
+            persistent._mas_windowreacts_database,
+            eventlabel="monika_twitter",
```
Comment (multimokia) [inline Monika After Story/game/zz_windowreacts.rpy]: Also, one issue. This label already exists in `script-topics.rpy`. You'll want to use a label which doesn't conflict. I recommend making this something like `monika_twiter_react` to differentiate.

**Comment by him157** [comment]
Done and done! Looking pretty good! It is a bit hard for me to get the right tone since English ain't my mother tongue. And you know what they say,thee heads are better than one,tho in this case we only got two,but good enough!

**Comment by multimokia** [comment]
Haha, I think you missed one of them, or marked a different one as resolved. Nice work though, looking forward to getting more and more of these.

**Comment by him157** [comment]
Well thank you my good sir,now I guess it is really done,can't wait to see what people can also come up to.

**Comment by multimokia** [comment]
Yep, but they're also flexible enough to do something like normal topics too, which is handy.

**Comment by him157** [comment]
and also just saying,the change request is done,I'm sure I done it all.
And it is just so incredible that you can spot out bugs and reused thing in lighting speed always.

**Comment by Rai99** [comment]
~~There's also a typo in the PR title, but close enough.~~

**Comment by him157** [comment]
_well look now we got three heads_

**Comment by multimokia** [comment]
One more thing actually. I'm going to modify all of the other windowreacts to use a set label format:
`mas_windowreacts_<name>`. So yours would become `mas_windowreacts_twitter` and `mas_windowreacts_monikatwitter` respectively. Would you mind changing both the labels accordingly?

Thanks

**Comment by him157** [comment]
No,no,not at all! You don't need to thank me,I'm changing it now!

**Comment by multimokia** [comment]
Before you do, label `mas_wrs_<name>`

**Comment by multimokia** [comment]
Awesome, thanks. I`ll give this a test go later today.

**Comment by him157** [comment]
Done and done,now just before you say,I also changed already existing one.

**Comment by multimokia** [comment]
Will need to do data transfer to new ev`s. Will do this later.

File: Monika After Story/game/zz_windowreacts.rpy [hunk 3]
```diff
@@ -379,18 +379,59 @@ init 5 python:
...
+            eventlabel="mas_wrs_monikatwitter",
+            category=['twitter', 'lilmonix3'],
+            rules={"notif-group": "Window Reactions", "skip alert": None},
+            show_in_idle=True
+        ),
+        code="WRS"
+    )
+
+label mas_wrs_monikatwitter:
+    call display_notif(m_name, "I see you're checking my twitter, [player].\nSee anything you want to talk about?",'Window Reactions')
```
Comment (Velius94) [inline Monika After Story/game/zz_windowreacts.rpy]: Capital T for Twitter.

Comment (Velius94) [inline Monika After Story/game/zz_windowreacts.rpy]: Capital T for Twitter.

**Comment by him157** [comment]
wow a lot of thing happened when I was asleep,yeah,it was night for me.

**Comment by multimokia** [comment]
Putting don't merge on this until #4398 gets in

**Comment by him157** [comment]
ok,I need some help,I'm just too blind to see what I done wrong and my eyes,it hurts a bit.

**Comment by multimokia** [comment]
I'll fix this later as it will have conflicts now

**Comment by him157** [comment]
thank you my good sir,it is 3:14 a.m. here now,so my head is probably a bit messed up,I can only think of the lullaby plays when holding my dear over 30 minutes with no music.

**Comment by multimokia** [comment]
Also, would be nice if you could add some more variants for your reactions, just so it's not the same every time.

**Comment by him157** [comment]
ok now my sir,can you help me check the phrasing and grammar.

**Comment by multimokia** [comment]
Uhh, didn't really want you to change the labels back. I undid those changes because of #4428

**Comment by him157** [comment]
oh,ok,give me a minute.

**Pull request merged**